python/ray/air/_internal/device_manager/npu.py

```diff
@@ -55,7 +55,7 @@ def get_devices(self) -> List[torch.device]:
 
             if len(npu_ids) > 0:
                 npu_visible_str = os.environ.get(
-                    ray_constants.NPU_RT_VISIBLE_DEVICES_ENV_VAR, ""
+                    ray_constants.ASCEND_RT_VISIBLE_DEVICES_ENV_VAR, ""
```

Comment:
To avoid duplication, we can remove from ray_constants.py and just import the relevant accelerator file to get the corresponding constant?

Reply:
This also makes sense. @edoakes does this make sense to you?

Reply:
yea this sounds better to me